### PR TITLE
[14.0][MIG] purchase_order_uninvoiced_amount

### PR DESCRIPTION
**Pull request by SurachetS**
standard migration

### CODE REVIEW

**Review by tonygalmiche — APPROVED** [review]
LGTM : Functional review only with runbot

image

File: purchase_order_uninvoiced_amount/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Tecnativa - Manuel Calero
+# Copyright 2020 Tecnativa - João Marques
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Purchase Order Univoiced Amount",
```
Comment (MeetKD) [inline purchase_order_uninvoiced_amount/__manifest__.py:6]: @SurachetS It would be great if you can add module summary.

**Review by MeetKD — CHANGES_REQUESTED** [review]

File: purchase_order_uninvoiced_amount/__manifest__.py (hunk 1 above)
Comment (SurachetS) [reply purchase_order_uninvoiced_amount/__manifest__.py:6]: @MeetKD Updated :)

**Comment by kittiu** [comment]
Code review, LGTM. :+1:

**Review by kittiu — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please include #1056

**Comment by pedrobaeza** [comment]
Better to cherry-pick a1e051e89a2ca3f90683bee777234f186b485ef8 that I have adapted to v13 as it's more similar to v14.

**Pull request merged**